src/api/app/controllers/person_controller.rb

```diff
@@ -237,7 +237,7 @@ def change_my_password
   end
 
   def change_password(login, password)
-    unless User.current
+    unless User.required
```

Comment:
this kind of control shouldn't be done as, normally with others authentication framework, with a ```before_filter```?

Reply:
that question doesn't make sense to me (syntactically). This code should use a different methology to check users, but what is your question exactly?